build.sbt

```diff
@@ -148,7 +149,21 @@ lazy val site = project
   )
   .dependsOn(kernel, githubActions, noPublish)
 
+lazy val unidoc = project
+  .in(file("unidoc"))
+  .enablePlugins(TypelevelUnidocPlugin)
+  .settings(
+    // Doesn't conflict w/ core artifact since this one won't have the sbt cross version suffix
+    // javadoc.io doesn't recognize sbt artifacts, so we must publish like this
+    name := "sbt-typelevel",
+    mimaPreviousArtifacts := Set.empty
+  )
+
 lazy val docs = project
   .in(file("mdocs"))
   .enablePlugins(TypelevelSitePlugin)
-  .settings(laikaConfig ~= { _.withRawContent })
+  .settings(
+    laikaConfig ~= { _.withRawContent },
+    tlSiteApiModule := Some((unidoc / projectID).value),
+    tlSiteApiPackage := Some("org.typelevel.sbt")
```

Comment:
This is a bit clunky.

Reply:
I have an idea to set `tlSiteApiModule := Some((unidoc / projectID).value)` automatically.

Reply:
A bit reflection-y but gets the job done :)

https://github.com/typelevel/sbt-typelevel/blob/0e013b49670667689f6ca43003fbda66ec38a330/sonatype/src/main/scala/org/typelevel/sbt/TypelevelUnidocPlugin.scala#L36-L37